jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java

```diff
@@ -83,16 +84,29 @@ public void testEntrypoint_nonDefaultAppRoot() throws IOException {
     Assert.assertEquals(
         "ENTRYPOINT [\"java\",\"-cp\",\"/resources:/classes:/libs/*\",\"MainClass\"]",
         getEntrypoint());
+    Assert.assertEquals("CMD []", getCmd());
   }
 
   @Test
-  public void testEntrypoint_defaultWebAppRoot() throws IOException {
+  public void testEntrypoint_inheritedEntrypoint() throws IOException {
     Mockito.when(containerParameters.getAppRoot()).thenReturn("/");
+    Mockito.when(containerParameters.getArgs()).thenCallRealMethod();
     project.getPluginManager().apply("war");
 
     task.generateDockerContext();
 
-    Assert.assertEquals("ENTRYPOINT [\"java\",\"-jar\",\"/jetty/start.jar\"]", getEntrypoint());
+    try {
+      getEntrypoint();
+      Assert.fail();
+    } catch (NoSuchElementException e) {
+      Assert.assertEquals("No value present", e.getMessage());
```

Comment:
If this is the generic message for `NoSuchElementException`, we can just replace this with `// pass`.